jobpostings/views.py

```diff
@@ -110,4 +117,41 @@ def get(self, request):
             return JsonResponse({"message":"SUCCESS", "result":result}, status=200)
 
         except KeyError:
-            return JsonResponse({"message" : "KEY_ERROR"}, status=400)
\ No newline at end of file
+            return JsonResponse({"message" : "KEY_ERROR"}, status=400)
+
+class PostingView(View):
+    @lose_authorization
+    def get(self, request, posting_id):
+        print(request)
+        user        = request.user
```

Comment:
```suggestion
        user = request.user
```
print 제거하고 = 간격 줄여 주세요.

Reply:
수정했습니다.